src/client/editor/SceneLoader.js

```diff
@@ -538,6 +541,9 @@ export async function loadScene(uri, addComponent, isRoot = true, ancestors) {
   }
 
   const sceneResponse = await fetch(url);
+  if (!sceneResponse.ok) {
+    throw url;
```

Comment:
It's generally bad practice to `throw` an object that is not an `Error`. Best to use a custom error object like `SceneLoadingError`, as I mentioned above.